repo/fsrepo/fsrepo_test.go

```diff
@@ -30,31 +30,7 @@ func TestInitIdempotence(t *testing.T) {
 func TestRemove(t *testing.T) {
 	t.Parallel()
 	path := testRepoPath("foo", t)
-	assert.Nil(Remove(path), t, "should be able to remove after closed")
-}
-
-func TestCannotRemoveIfOpen(t *testing.T) {
```

Comment:
The test is about the process *itself* holding it open. Nobody does that, and it just sounds like a very bad bit of programming. `Remove` did not check for other processes before this commit, and isn't checking now -- the calling `ipfs init` does that.